packages/datadog-instrumentations/src/fastify.js

```diff
@@ -133,6 +134,17 @@ function preValidation (request, reply, done) {
       if (abortController.signal.aborted) return
     }
 
+    if (pathParamsReadCh.hasSubscribers && request.params) {
+      const abortController = new AbortController()
```

Comment:
Creating new AbortControllers is quite expensive. Could we just reuse an already defined controller?
I believe it's possible to unify the up to three controllers to a single one.